src/Client/pages/NewBook.fs

```diff
@@ -58,9 +58,9 @@ let update (msg:Msg) model : Model*Cmd<Msg> =
                 TitleErrorText = model.NewBook.ValidateTitle()
                 AuthorsErrorText = model.NewBook.ValidateAuthors()
                 LinkErrorText = model.NewBook.ValidateLink() }
-        validated, 
+        validated,
             if model.NewBook.Validate() then
-                Cmd.ofMsg AddBook
+                Cmd.OfFunc.result AddBook
```

Comment:
urgs!?

Reply:
A long discussion leads to this changes: https://github.com/elmish/elmish/issues/166